py3-pgcli.yaml

```diff
@@ -36,13 +37,85 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: 46fe654cd16a3af837639dc2c8be60d1b7a038c7
 
-  - name: Python Build
-    runs: python setup.py build
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      runtime:
+        - py${{range.key}}-configobj
+        - py${{range.key}}-typing-extensions
+        - py${{range.key}}-click
+        - py${{range.key}}-sqlparse
+        - py${{range.key}}-cli-helpers
+        - py${{range.key}}-prompt-toolkit
+        - py${{range.key}}-pygments
+        - py${{range.key}}-pgspecial
+        - py${{range.key}}-pendulum
+        - py${{range.key}}-psycopg
+        - py${{range.key}}-setproctitle
+    pipeline:
+      - name: Python Build
```

Comment:
i think `uses: py/pip-build-install` will work here. then its just one section rather than 2 and more things use it than the other.